rosys/analysis/timelapse_recorder.py

```diff
@@ -70,8 +72,12 @@ async def save(self, image: RosysImage) -> None:
                                   self._notifications.pop(0) if self._notifications else [],
                                   self.frame_info_builder(image))
 
-    async def compress_video(self) -> None:
-        """Creates a video from the captured images"""
+    def compress_video(self) -> Task:
+        """Creates a video from the captured images
+        Note: this method starts a background task and returns immediately."""
```

Comment:
```suggestion
        """Create a video from the captured images.
        
        Note: This method starts a background task and returns immediately.
        """
```
Following PEP257 for multi-line docstrings.